setup.py

```diff
@@ -8,144 +8,310 @@
+
+    for filename in sorted(shaderfiles):
+        shaderfile = os.path.join(shaderdir, filename)
+        if not os.path.exists(shaderfile):
+            shaderfile = os.path.join(shaderdir2, filename)
+
+        with open(shaderfile, "r") as handle:
+            contents = handle.read()
+
+        if True:
+            outputfile.write('"%s", ""\n' % (filename))
+
+            for line in contents.splitlines():
+                line = line.strip()
+
+                # skip blank lines and obvious comments
+                if not line or line.startswith("//") and not "*/" in line:
+                    continue
+
+                # write line, quoted, escaped and with a line feed
+                outputfile.write(
+                    '"%s\\n"\n' % line.replace("\\", "\\\\").replace('"', r"\"")
+                )
+
+                # include and ifdef dependencies
+                if line.startswith("#include"):
+                    include_deps[line.split()[1]].add(filename)
+                elif line.startswith("#ifdef") or line.startswith("#ifndef"):
+                    ifdef_deps[line.split()[1]].add(filename)
+
+            outputfile.write(",\n")
+
+    outputfile.write("0};\n")
+
+    # include and ifdef dependencies
+    for varname, deps in [("_include_deps", include_deps), ("_ifdef_deps", ifdef_deps)]:
+        outputheader.write("extern const char * %s[];\n" % varname)
+        outputfile.write("const char * %s[] = {\n" % varname)
+        for name, itemdeps in deps.items():
+            outputfile.write('"%s", "%s", 0,\n' % (name, '", "'.join(sorted(itemdeps))))
+        outputfile.write("0};\n")
+
+    outputheader.close()
+    outputfile.close()
+
+
+def create_buildinfo(outputdir, pymoldir="."):
+    try:
+        sha = (
+            Popen(["git", "rev-parse", "HEAD"], cwd=pymoldir, stdout=PIPE)
+            .stdout.read()
+            .strip()
+            .decode()
+        )
+    except OSError:
+        sha = ""
+
+    with openw(os.path.join(outputdir, "PyMOLBuildInfo.h")) as out:
+        print(
+            """
+#define _PyMOL_BUILD_DATE %d
+#define _PYMOL_BUILD_GIT_SHA "%s"
+        """
+            % (time.time(), sha),
+            file=out,
+        )
 
-WIN = sys.platform.startswith('win')
-MAC = sys.platform.startswith('darwin')
 
 # handle extra arguments
 
+def str2bool(v: str):
+    if v.lower() == "true":
+        return True
+    elif v.lower() == "false":
+        return False
+    else:
+        raise argparse.ArgumentTypeError('Boolean value expected.')
+
 
 class options:
     osx_frameworks = True
-    jobs = int(os.getenv('JOBS', 0))
+    jobs = int(os.getenv("JOBS", 0))
     no_libxml = False
     no_glut = True
-    use_msgpackc = 'guess'
+    use_msgpackc = "guess"
     testing = False
     openvr = False
-    use_openmp = 'no' if MAC else 'yes'
-    use_vtkm = 'no'
-    vmd_plugins = True
+    use_openmp = "no" if MAC else "yes"
+    use_vtkm = "no"
+    vmd_plugins = False
```

Comment:
This should be left at `True` by default for tests to pass (and so ofc by default users can use this to open several file types).

Reply:
Oops, I didn't even notice when it changed. Fixed